background.js

```diff
@@ -2,6 +2,17 @@
 
 import { addToArchiveBox } from "./utils.js";
 
+class Entry {
```

Comment:
```suggestion
class Snapshot {
```

Can we change this to `Snapshot` and `snapshots` everywhere to keep the naming consistent with the server.